openedx/core/djangoapps/content/learning_sequences/tests/test_views.py

```diff
@@ -56,15 +57,13 @@ def setUp(self):
     def url_for(cls, course_key):
         return '/api/learning_sequences/v1/course_outline/{}'.format(course_key)
 
-    def test_student_access_denied(self):
+    def test_student_access(self):
         """
-        For now, make sure you need staff access bits to use the API.
-
-        This is a temporary safeguard until the API is more complete.
+        Check whether a student can access the course outline api
         """
         self.client.login(username='student', password='student_pass')
         result = self.client.get(self.course_url)
-        assert result.status_code == 403
+        assert result.status_code == 200
```

Comment:
Does it make sense to have a test for student access being denied when the `course_visibility` is `CourseVisibilities.PRIVATE`?

Reply:
@pcockwell Not sure about this.

Reply:
Generally I tend to think more tests are better, though I will let @ormsbee weigh in on this one

Reply:
I think it makes sense to add that test.